cmd/jimmctl/cmd/group.go

```diff
@@ -21,10 +20,6 @@ import (
 )
 
 const (
-	groupDoc = `
-The group command enables group management for jimm
-`
-
 	addGroupDoc = `
```

Comment:
I think... Now this is going top level, and could become confusing to juju users let's get super explicit about these commands. And also format it somewhat similar to juju's existing commands like add-model:
```
❯ jj add-model -h
Usage: juju add-model [options] <model name> [cloud|region|(cloud/region)]

Summary:
Adds a workload model.
```
They don't say "the X command does Y", they just summarise it. Or should we change wording in a follow up?

Reply:
Good point, fixed for all the commands to just `Adds a group.` from `The add command adds group to jimm.`